pom.xml

```diff
@@ -12,9 +12,9 @@
 	</parent>
 	<properties>
 		<java.version>1.8</java.version>
-		<spring-kafka.version>2.0.0.RELEASE</spring-kafka.version>
-		<kafka.version>0.11.0.0</kafka.version>
-		<spring-integration-kafka.version>3.0.0.M2</spring-integration-kafka.version>
+		<spring-kafka.version>2.1.0.RC1</spring-kafka.version>
```

Comment:
Soby, please note that the 2.0.0.RELEASE renames the `KStreamBuilderFactoryBean` to align with renaming in the kafka 1.0.0 client: https://github.com/spring-projects/spring-kafka/pull/500